scripts/ninja.js

```diff
@@ -861,7 +861,7 @@ async function runtimeNinja(devmode = true) {
   var externalDeps = devmode ? [compilerTarget] : [];
   var ninjaOutput = devmode
```

Comment:
can we keep this file unchanged for this PR

Reply:
`env.ninja` is not used and maintained for a while, we may consider removing it and add the corresponding one later.

Reply:
I reverted changes to ninja.js